src-ghc/Pact/Types/Command.hs

```diff
@@ -97,23 +97,22 @@ instance (FromJSON a) => FromJSON (Command a) where
 instance NFData a => NFData (Command a)
 
 
+requestKeyHash :: H.Blake2b_512
+requestKeyHash = H.Blake2b_512
+
 
 mkCommand :: (ToJSON m, ToJSON c) =>
              [SomeKeyPair] -> m ->
-             Text -> PactRPC c -> Command ByteString
+             Text -> PactRPC c -> IO (Command ByteString)
```

Comment:
I needed to make this IO because signing in ECDSA adds some randomness, which ED25519 does not do.